ArduPlane/quadplane.h

```diff
@@ -80,6 +80,7 @@ class QuadPlane
     
     // update transition handling
     void update(void);
+    bool in_forced_transition;
```

Comment:
This should be here:

https://github.com/ArduPilot/ardupilot/blob/f6fa676411dc7318224d2f6b92c309bd7aca2bd9/ArduPlane/transition.h#L112

You can then use it directly as `in_forced_transition` rather than `quadplane.in_forced_transition`

Reply:
done